ui/raidboss/data/02-arr/raid/t12.txt

```diff
@@ -10,7 +10,7 @@ hideall "Scorched Pinion"
 
 ### Phase 1: Blackfire/Whitefire
 0.0 "Start"
-0.0 "--sync--" sync /:Engage!/ window 0,1
+0.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 0,1
 2.5 "--sync--" sync / 1[56]:[^:]*:Phoenix:368:/ window 3,0
```

Comment:
I think in general (here and other timelines) we could also remove the initial auto (which is often inconsistent) and add a larger sync on the first ability.